src/System.Private.Uri/tests/FunctionalTests/UriTests.cs

```diff
@@ -731,13 +734,16 @@ public static void TestIsWellFormedOriginalString()
             b = uri.IsWellFormedOriginalString();
             Assert.False(b);
 
-            uri = new Uri(@"c:\\directory\filename");
-            b = uri.IsWellFormedOriginalString();
-            Assert.False(b);
+            if (s_IsWindowsSystem)
+            {
+                uri = new Uri(@"c:\\directory\filename");
+                b = uri.IsWellFormedOriginalString();
+                Assert.False(b);
 
-            uri = new Uri(@"file://c:/directory/filename");
-            b = uri.IsWellFormedOriginalString();
-            Assert.False(b);
+                uri = new Uri(@"file://c:/directory/filename");
+                b = uri.IsWellFormedOriginalString();
+                Assert.False(b);
+            }
```

Comment:
Can/should we validate the inverse on Unix, that these aren't recognized as file paths?

Though related to some of my previous comments, I also wonder whether we can/should try to make the implementation recognize, on unix, unix file paths in addition to rather than instead of windows ones.